FacialRecog.py

```diff
@@ -85,38 +92,74 @@ def scan_for_known_people(known_people_folder):
                 face_encodings.append(single_encoding[0])
 
                 # write to file, for performance reasons, so as to not calculate all the faces each time
-                encodedfile = np.save((os.path.join(known_people_folder, "PreEncoded"+filename) + ".npy"), single_encoding[0])
+                encodedfile = np.save((os.path.join(known_people_folder, "PreEncoded" + filename) + ".npy"),
+                                      single_encoding[0])
                 print("DEBUG: saved to document", filename)
                 print("DEBUG: saved to document", encodedfile)
 
     return names, face_encodings
 
+"""
+Gets data from KNOWN db and detects if facial encodings are the same
+"""
+def scan_for_known_people_from_db(npy_list: str):
+    all_encodings = []
+    data = get_all_known("not_api_call")
+
+    for i in data:
+        # decode b64 np array and save that into a list
+        foo = i['npy'].strip('][').split(', ')
+        for j in range(len(foo)):
+            foo[j] = float(foo[j])
+        all_encodings.append(foo)
+
+    all_encodings = np.array(all_encodings)
+    npy_array = np.array(npy_list)
+    detected_faces = face_recognition.compare_faces(all_encodings, npy_array)
+
+    return detected_faces
```

Comment:
for this to be able to be plugged into RecogScript w/ minimal refactoring we need to to return known_names, known_encodings

Reply:
wait why would we return the encodings? detected_faces is just a true/false list

Reply:
because it's gonna be plugged in where `scan_for_known_people` is currently in RecogScript

![image](https://user-images.githubusercontent.com/31379789/75295054-85188b80-57ef-11ea-9f5e-5f7e77feab37.png)

we already have code to do the comparing, we just need the faces and names to compare against